workflow/rules/capacity_factors.smk

```diff
@@ -0,0 +1,31 @@
+if config["layout"] == "raster":
```

Comment:
This could be handled without the need for a top-level configuration item. If you pass a .tif file as layout, you get raster, if you pass a .csv file, you get point. E.g, you have a single rule which calls a simple wrapper script that decides what to do based on the file ending of `input.layout`, calling one or the other function for further processing. Everything else seems to be 100% the same across these two rules in here.